Makefile

```diff
@@ -201,9 +205,14 @@ license-cache: bin/licensei ## Generate license cache
 .PHONY: check
 check: lint test helm-lint ## Run tests and linters
 
+TIDYGOMODULES = $(addprefix tidy-, $(GOMODULES))
+
+.PHONY: $(TIDYGOMODULES)
+$(TIDYGOMODULES):
+	cd $(dir $(@:tidy-%=%)) && go mod tidy
```

Comment:
```suggestion
	cd $(dir $(@:tidy-%=%)) && go mod tidy -go=1.20
```

Reply:
What would be even better if the GO version for `go mod tidy` would be passed as a variable and determined based on the `go ...` directive in the `go.mod` files.

Reply:
> If the -go flag is set, go mod tidy will update the [go directive](https://go.dev/ref/mod#go-mod-file-go) to the indicated version
 
It might be better to have this controlled in the Makefile, then it can force all our modules to use the same go version instead of maintaining it per go.mod

Reply:
One way to get the go version from go.mod:

Reply:
> > If the -go flag is set, go mod tidy will update the [go directive](https://go.dev/ref/mod#go-mod-file-go) to the indicated version
> 
> It might be better to have this controlled in the Makefile, then it can force all our modules to use the same go version instead of maintaining it per go.mod

Agreed.

Reply:
thanks for reviewing again 😄 please check the last commit and let me know what you think